README.dev.md

```diff
@@ -42,9 +42,9 @@ coverage report
 
 `coverage` can also generate output in HTML and other formats; see `coverage help` for more information.
 
-## Linting
+## Linting and Formatting
 
-We use [prospector](https://pypi.org/project/prospector/) with pyroma for linting. For running it locally, use `prospector` or `prospector <filepath_or_folderpath>` for specific files/folders.
+We use [ruff](https://docs.astral.sh/ruff/) for linting, sorting imports and formatting code. The configurations of `ruff` are set in [pyproject.toml](pyproject.toml) file. If you are using VS code, you can activate the [Ruff extension](charliermarsh.ruff) to automatically format the code and flag linting issues upon saving. Otherwise, please check both linting (`ruff .`) and formatting (`ruff format .`, this automatically formats the entire code base) before requesting a review.
```

Comment:
Add the correct link for project.toml file

Reply:
The link works like this.